src/index.test.ts

```diff
@@ -101,12 +107,32 @@ describe('Stripe module loader', () => {
       const {loadStripe} = require('./index');
       const stripePromise = loadStripe('pk_test_foo');
 
-      return Promise.resolve().then(() => {
+      return Promise.resolve().then(async () => {
```

Comment:
(2) Can we not mix the `async/wait` and `.then` styles? (here and below)

Reply:
I guess we could promise chain, but we need to wait for the `stripePromise` rejection before testing `console.warn`